projects/ngx-wig/src/lib/ngx-wig.module.ts

```diff
@@ -0,0 +1,54 @@
+import { NgModule, CUSTOM_ELEMENTS_SCHEMA, NO_ERRORS_SCHEMA, ModuleWithProviders, InjectionToken } from '@angular/core';
+import { NgxWigComponent } from './ngx-wig.component';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { BrowserModule } from '@angular/platform-browser';
+import { TButtonLibrary, DEFAULT_LIBRARY_BUTTONS, BUTTONS } from './config';
+
+export function getWindowObject(): Window {
+  return window;
+}
+
+@NgModule({
+  declarations: [
+    NgxWigComponent
+  ],
+  imports: [
+    BrowserModule,
+    FormsModule,
+    ReactiveFormsModule,
```

Comment:
Should we let the consumer of the library control this?

Reply:
I thought that we are using it, but it looks like you are right, removed